### ISSUE TITLE
Large log file fills up hard drive space

### ISSUE THREAD

**Issue by oodg** [Tunnelblick/Tunnelblick#739]
Been using Tunnelblick for around 3 hours, after which I got errors because Tunnelblick created a 230 GB log file which consisting of following line repeating  every 0.1 ms:

```
1670619083.740469 1000021 read from TUN/TAP : Input/output error (code=5)
1670619083.740480 1000021 read from TUN/TAP : Input/output error (code=5)
1670619083.740490 1000021 read from TUN/TAP : Input/output error (code=5)
1670619083.740501 1000021 read from TUN/TAP : Input/output error (code=5)
1670619083.740512 1000021 read from TUN/TAP : Input/output error (code=5)
1670619083.740522 1000021 read from TUN/TAP : Input/output error (code=5)
1670619083.740533 1000021 read from TUN/TAP : Input/output error (code=5)
1670619083.740543 1000021 read from TUN/TAP : Input/output error (code=5)
1670619083.740554 1000021 read from TUN/TAP : Input/output error (code=5)
1670619083.740564 1000021 read from TUN/TAP : Input/output error (code=5)
1670619083.740575 1000021 read from TUN/TAP : Input/output error (code=5)
1670619083.740585 1000021 read from TUN/TAP : Input/output error (code=5)
1670619083.740596 1000021 read from TUN/TAP : Input/output error (code=5)
1670619083.740607 1000021 read from TUN/TAP : Input/output error (code=5)
```

**Comment by oodg** [comment]
> The VPN is working even with this error? I've never seen that!

It does!

> If so, just disable logging: Tunnelblick >> VPN Details >> Configurations >> Settings >> VPN log level.

Will do, thank you.:)

**Comment by jkbullard** [comment]
How do you know it is working? Does your IP address change, or do you access servers on the VPN's network that you can't otherwise, or is there some other indicator?

**Comment by oodg** [comment]
> How do you know it is working? Does your IP address change, or do you access servers on the VPN's network that you can't otherwise, or is there some other indicator?

I remotely connected to my local computer at home, so I'm sure it was connected.
 I am trying to reproduce the error, but I can't get it to appear. Once I am able to reproduce it I will report back!

**Comment by jkbullard** [comment]
Thanks, @oodg, for bringing up this problem. It is fixed in Tunnelblick 4.0.0beta05, released yesterday:

- The OpenVPN log file is erased whenever it becomes 200 MB long. The limit can be modified by setting the "maximumOpenvpnLogSize" preference, which is the limit expressed as a number of bytes and which can range from 1 MB to 1 TB (1,000,000 to 1,000,000,000,000).

Note that the log throttling introduced in a10ff83f does not affect the OpenVPN log _file_; it only affects what is shown on the log display in Tunnelblick's "VPN Details" window.
